### PR TITLE
Pomodoro

### PR DESCRIPTION
**Pull request by Hamza-777**
Implemented pomodoro for the pomodoro app.
Live: https://snazzy-kringle-b20178.netlify.app/

### CODE REVIEW

File: src/Components/Pages/TodoPage.jsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        minutes: 0,
+        seconds: 0
+    });
+    const [ timerOn, setTimerOn ] = useState(false);
+    const [ colorId, setColorId ] = useState('');
+
+    const { minutes, seconds } = timer;
+
+    useEffect(() => {
+        setCurrentTodo({...todos.todos.filter(todo => todo.id === id )[0]});
```
Comment (Maianki) [inline src/Components/Pages/TodoPage.jsx]: you can destructure todos above when you are destructuring it in line 14. So you don't have to do todos.todos.filter. 
Basically you can replace ```const { todos } = useTodos();``` to ```const { todos:{todos} } = useTodos();```

**Review by Maianki — APPROVED** [review]
Small PR nothing much to suggest. You have a good fold structure and the names of components and files are meaningful. Has added one suggestion do give it a look.

File: src/Components/Pages/TodoPage.jsx (hunk 1 above)
Comment (Hamza-777) [reply src/Components/Pages/TodoPage.jsx]: Thanks for the suggestion,  will make those changes.

File: src/Components/Pages/TodoPage.jsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                    1000
+                );
+                document.title = `${minutes}m : ${seconds < 10 ? 0 : ''}${seconds}s || POMODORO`;
+            } else {
+                setTimeout(
+                    () => setTimer({ ...timer, seconds: seconds - 1 }),
+                    1000
+                );
+                document.title = `${minutes}m : ${seconds < 10 ? 0 : ''}${seconds}s || POMODORO`;
+            }
```
Comment (animeshgarai09) [inline src/Components/Pages/TodoPage.jsx:55]: You can add a Cleanup function for setTimeOut.
if the component is unmounted then you can use clearTimeout as a cleanup function for useEffect

**Review by animeshgarai09 — APPROVED** [review]
Rest of the code looks good.

File: src/Components/Pages/TodoPage.jsx (hunk 2 above)
Comment (Hamza-777) [reply src/Components/Pages/TodoPage.jsx:55]: Thanks for the suggestion but I read somewhere that you don't need a clearTimeout unless you dont want the timeout to be cleared before the timeout even begins

**Pull request merged**